src/olympia/bandwagon/views.py

```diff
@@ -703,14 +704,35 @@ def get_addons_queryset(self):
         return qs.order_by(sort)[:limit]
 
 
+class TranslationAwareOrderingAliasFilter(OrderingAliasFilter):
```

Comment:
Is this something that is generally useful outside of Collections?  (we sort by name for Addons too?)  If so could/should be moved to `api/filters.py`

Reply:
The add-ons endpoints are using ElasticSearch directly though instead of the database. So, for now this isn't generic unfortunately and I'm not sure it's easy to do in a more generic way. I'm happy to open a code-quality issue to implement a more abstract way but I guess it's a bit overkill for this bugfix.

Reply:
fair point